src/cuga/backend/cuga_graph/policy/tool_guard/tool_guard_runtime.py

```
@@ -0,0 +1,764 @@
+        try:
+            logger.info(f"Loading runtime domain for app '{app_name}'...")
+            runtime_domain = self._load_runtime_domain(app_name)
+            self._runtime_domains_by_app[app_name] = runtime_domain
+            
+            logger.info(f"Building runtime for app '{app_name}'...")
+            runtime = self._build_runtime(app_name)
+            self._runtimes_by_app[app_name] = runtime
+            
+            logger.info(f"✅ Runtime initialized for app '{app_name}'")
+            return runtime
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)